packages/analytics/analytics-geo-map/src/locales/en.json

```diff
@@ -0,0 +1,6 @@
+{
+  "header": {
```

Comment:
Do you not need i18n? If no? You can remove this

Reply:
added it